### PR TITLE
adjust sample config to random config

### PR DESCRIPTION
**Pull request by mwiktorin**

### CODE REVIEW

File: src/main/config/sample-scenario.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
           "sales_price": 15
         },
         {
-          "id": "bagel",
+          "guid": "bagel",
```
Comment (moriarty) [inline src/main/config/sample-scenario.json]: Does "bagel" really fit as a "guid": Globally Unique ID?

File: src/main/config/sample-scenario.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
       ],
       "products": [
         {
-          "id": "bread",
+          "guid": "bread",
```
Comment (moriarty) [inline src/main/config/sample-scenario.json]: Does "bread" really fit as a "guid"?

https://blog.codinghorror.com/primary-keys-ids-versus-guids/

File: src/main/config/sample-scenario.json [hunk 3]
```diff
@@ -106,7 +106,7 @@
   "orders": [
     {
       "guid": "order-001",
-      "customer_id": "Citadel Cafe",
+      "customer_id": "customer-002",
```
Comment (moriarty) [inline src/main/config/sample-scenario.json]: I don't see why this change is required.

**Comment by argenos** [comment]
huh, I thought I had deleted the file with the intention of making everyone switch and use the random-scenario instead. 

@moriarty there was a small discussion in class about guids for products, and we decided to leave it as is for now. I do think that for the sake of consistency product-001 is better. 
We can accept the merge request and delete it for the next class/assignment.

**Comment by mwiktorin** [comment]
I just adjusted the sample to the random config. You should ask all these questions to the random config too, because I used it as a template for these changes. With these changes they are same so one program can parse both files.
